### PR TITLE
Processing Google Forms with XSOAR - Article

### PR DESCRIPTION
**Pull request by guyfreund**
## Status
In Progress

## Description
An article on a specific use-case of processing google forms with XSOAR

### CODE REVIEW

**Review by ShahafBenYakir — APPROVED** [review]
Nice
See my comments

File: content-repo/extra-docs/articles/processing_google_forms_via_webhook.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+id: google-forms-cortex-xsoar
+title: Processing your Google Form responses in Cortex XSOAR via webhook
```
Comment (glicht) [inline content-repo/extra-docs/articles/processing_google_forms_via_webhook.md:3]: Let's make this shorter:
```suggestion
title: Processing Google Form Responses via a Webhook 
```

**Review by glicht — COMMENTED** [review]
Looks good. A few improvement suggestions:

* Separate to 2 sections: 
  - Generic Webhook Configuration
  - Google Forms Configuration
 * Add note that the webhook needs to be accessible from the internet. Customers may need to use follow: https://xsoar.pan.dev/docs/reference/articles/long-running-invoke or possibly need to run the webhook integration on an engine which is accessible from the internet.
 * Mention the need to set username/password in webhook config.
 * Mention the use of `store_samples` in webhook to be able to create a mapper. Also, add a screenshot of a sample mapper showing an event from Google Forms.
 * Add a screen shots of the Google Forms Editor with the Script and steps 8 and 9

File: content-repo/extra-docs/articles/processing_google_forms_via_webhook.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+3. Create a designated playbook to run when the incident is created.
+
+4. Download the [Generic Webhook Pack](https://xsoar.pan.dev/marketplace/details/GenericWebhook) and configure an instance of the integration. For example, you can call the instance: *GenericWebhookForm*.
+
+5. Create a Google Apps Script for the selected form. See [Creating a project from Google Docs, Sheets, or Forms](https://developers.google.com/apps-script/guides/projects#creating_a_project_from_google_docs_sheets_or_forms) for more information.
+
+6. Add the `https://www.googleapis.com/auth/script.external_request` scope to your **appsscript.json** file. See [this](https://developers.google.com/apps-script/concepts/scopes) for more information.
+
+7. Create a new file in the script editor with the following generic code. (Replace the username, password, and URL with the values from the Generic Webhook integration):
+  ```
```
Comment (glicht) [inline content-repo/extra-docs/articles/processing_google_forms_via_webhook.md:30]: Set the content type for rich code
```suggestion
  ```javascript
```

File: content-repo/extra-docs/articles/processing_google_forms_via_webhook.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      var key = itemResponse.getItem().getTitle();
+      var value = itemResponse.getResponse();
+      formData[key] = value;
+      Logger.log('Response to the question "%s" was "%s"', key, value);
+    }
+    sendPostRequestToXSOAR(formData);
+  }
+  function sendPostRequestToXSOAR(formData) {
+    var body = {name: "New response to the form $FORM_NAME", raw_json: formData};
+    var username = $USERNAME;
```
Comment (glicht) [inline content-repo/extra-docs/articles/processing_google_forms_via_webhook.md:46]: where is $USERNAME coming from? Is there need to replace this in the code or it is coming form an env variable? If there is need to replace put it as `<YOUR_USERNAME>`

**Review by glicht — APPROVED** [review]
Looks good. Added some small formatting. See my last commit.

**Pull request merged**